React/Views/RCTRefreshControl.m

```diff
@@ -18,12 +18,17 @@ @implementation RCTRefreshControl {
   UIColor *_titleColor;
 }
 
+static unsigned short defaultHeight;
```

Comment:
Yes I think it is better even if it will be the same value for each instance. RefreshControl is not a component that has many instances so the memory usage is negligible. Also small detail but maybe `initialHeight` would be a better name.